src/index.html

```diff
@@ -11,7 +11,135 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="form"
+    >
+      <fieldset name="fieldset" class="fieldset">
+        <legend name="legend">Personal information</legend>
+        <label class="form-field" name="label">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="input"
+          >
+        </label>
+        <label class="form-field" name="input">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="input"
+          >
+        </label>
+        <label class="form-field" name="input">
+            How old are You?
+          <input
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            name="input"
+          >
+        </label>
+        <label class="form-field" name="input">Full date of birth:
+          <input type="date" name="input">
+        </label>
```

Comment:
place children from the new line 
```suggestion
        <label class="form-field" name="input">
          Full date of birth:
          <input type="date" name="input">
        </label>
```